### PR TITLE
Updating build process to run in heroku with babel

### PR DESCRIPTION
**Pull request by thismarcoantonio**

### CODE REVIEW

File: server/src/routes/auth.js [hunk 1]
```diff
@@ -38,4 +38,12 @@ export default (app) => {
 
     return res.sendStatus(401);
   });
+
+  app.get('/', (req, res) => {
```
Comment (rntgspr) [inline server/src/routes/auth.js:42]: routes should have incremental order/organization?

Comment (thismarcoantonio) [reply server/src/routes/auth.js:42]: '/' route is just a placeholder, it'll be removed later

Comment (rntgspr) [reply server/src/routes/auth.js:42]: Coool, I can handle it!

Comment (thismarcoantonio) [reply server/src/routes/auth.js:42]: Merged #9

**Pull request merged**